src/app/shared/services/apis/gb3/gb3-topics.service.ts

```diff
@@ -369,14 +370,20 @@ export class Gb3TopicsService extends Gb3ApiService {
     };
   }
 
-  private createFeatureInfoFieldValue(field: InfoFeatureField): string | LinkObject | null {
-    if ('link' in field) {
+  private createFeatureInfoFieldValue(field: InfoFeatureField): string | LinkObject | Image | null {
+    if (field.value === null) {
+      return null;
+    }
+    if (typeof field.value === 'number') {
+      return field.value.toString();
+    }
+    if (typeof field.value === 'string' || 'alt' in field.value) {
```

Comment:
That one is quite implicit. Maybe add a comment explaining that you check for string OR image, and if both fail, it must be a URL.